src/components/cvaa-overlay/cvaa-overlay.tsx

```diff
@@ -138,7 +138,6 @@ class CVAAOverlay extends Component<any, any> {
         addAccessibleChild={this.props.addAccessibleChild}
         onClose={props.onClose}
         type="cvaa"
-        label={props.cvvaDialogText}
```

Comment:
why remove it ?

Reply:
The label was passed as a prop just for the aria-label property that was replaced.